### PR TITLE
[Feature](Job)Job tasks support the choice of persistence or storage in memory

### PR DESCRIPTION
**Pull request by CalvinKirs**
### abstract
Jobs of the MTMV category do not require persistence, so we store them in memory

### update
- Job tasks support the choice of persistence or storage in memory 

- Show Jobs persisted to memory are only allowed on the master node. 
- Fix some task deletion logic.

### CODE REVIEW

**Comment by CalvinKirs** [comment]
run buildall

**Comment by CalvinKirs** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/scheduler/manager/JobTaskManager.java [hunk 1]
```diff
@@ -70,13 +70,18 @@ public static void clearPrepareTaskByJobId(Long jobId) {
...
                 .computeIfAbsent(jobTask.getJobId(), k -> new ConcurrentLinkedQueue<>());
         jobTasks.add(jobTask);
         if (jobTasks.size() > TASK_MAX_NUM) {
             JobTask oldTask = jobTasks.poll();
-            Env.getCurrentEnv().getEditLog().logDeleteJobTask(oldTask);
+            if (persist) {
+                Env.getCurrentEnv().getEditLog().logDeleteJobTask(oldTask);
+            }
+        }
+        if (!persist) {
```
Comment (zddr) [inline fe/fe-core/src/main/java/org/apache/doris/scheduler/manager/JobTaskManager.java:83]: if (persist) {
Env.getCurrentEnv().getEditLog().logCreateJobTask(jobTask);
}

**Comment by CalvinKirs** [comment]
run buildall

**Comment by CalvinKirs** [comment]
run buildall

**Review by zddr — APPROVED** [review]
LGTM

**Review by hf200012 — APPROVED** [review]

**Pull request merged**